### ISSUE TITLE
[Usage]: How to get the generated_tokens before current_token in the new logits processor.

### ISSUE THREAD

**Issue by johnking0099** [vllm-project/vllm#23160]
### Your current environment

```text
Not necessary.
```

### How would you like to use vllm

In the version 0.10.1 (just released), I see V1 engine supports logits processor now. But I cannot find the way to find all the tokens generated so far. Is it possible to get it?

### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by johnking0099** [comment]
Does anyone know the answer?
